libs/vkd3d/command.c

```diff
@@ -16533,12 +16535,28 @@ static void STDMETHODCALLTYPE d3d12_command_list_CopyRaytracingAccelerationStruc
 
     d3d12_command_list_end_current_render_pass(list, true);
     d3d12_command_list_end_transfer_batch(list);
-    vkd3d_acceleration_structure_copy(list, dst_data, src_data, mode);
 
-    VKD3D_BREADCRUMB_AUX64(dst_data);
-    VKD3D_BREADCRUMB_AUX64(src_data);
-    VKD3D_BREADCRUMB_AUX32(mode);
-    VKD3D_BREADCRUMB_COMMAND(COPY_RTAS);
+    vkd3d_va_map_try_read_rtas(&list->device->memory_allocator.va_map,
+            list->device, src_data, &src_as, &src_omm);
+
+    if (src_as)
+    {
+        vkd3d_acceleration_structure_copy(list, dst_data, src_as, mode);
+        VKD3D_BREADCRUMB_AUX64(dst_data);
+        VKD3D_BREADCRUMB_AUX64(src_data);
+        VKD3D_BREADCRUMB_AUX32(mode);
+        VKD3D_BREADCRUMB_COMMAND(COPY_RTAS);
+    }
+    else if (src_omm)
+    {
+        vkd3d_opacity_micromap_copy(list, dst_data, src_omm, mode);
+        VKD3D_BREADCRUMB_AUX64(dst_data);
+        VKD3D_BREADCRUMB_AUX64(src_data);
+        VKD3D_BREADCRUMB_AUX32(mode);
+        VKD3D_BREADCRUMB_COMMAND(COPY_OMM);
+    }
+    else
+        ERR("Invalid src address #%"PRIx64" for RTAS / OMM copy.\n", src_data);
```

Comment:
Same concern. If you cannot find anything resort to RTAS placement and flag a FIXME.